package-lock.json

```diff
@@ -18,11 +18,11 @@
         "@types/react": "^18.0.28",
         "@types/react-dom": "^18.0.11",
         "@vitejs/plugin-react": "^3.1.0",
+        "@vitest/coverage-c8": "^0.30.1",
         "autoprefixer": "^10.4.14",
```

Comment:
@Christa22 solve the conflicts in this `package-lock.json` file. To prevent conflicts in the package-lock.json file, you can add it to the .gitignore file in your project.